CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ### Important
 
-Version 0.18.0 of `pulumi-aws` is now based on v2.2.0 of the AWS Terraform Provider, which has a variety of breaking changes from the previous version.
+- Version 0.18.0 of `pulumi-aws` is now based on v2.2.0 of the AWS Terraform Provider, which has a variety of breaking changes from the previous version.
```

Comment:
Should we add a link to the details of the Terraform provider breaking changes?

Reply:
Yes - I'll do that when I prepare the changelog for release.